### DISCUSSION TITLE
Primeng editor converting non http urls to about:blank

### DISCUSSION THREAD

**Discussion by prateekjain123** [primefaces/community#1445] [PrimeNG] [answered]
Primeng editor converts non http urls to about:blank.

I want it to except all different urls or any option to whitelist few urls? If there is any way to handle in text change or by using custom link in editor's toolbar. Please let me know.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

We're using Quill in the editor component and we can't interfere with Quill's internal logic. We're just wrapping the Quill editor with some easy-to-use visual options. We'll remove Quill in the future since it's an outdated library. The best action would be the replace the Quill editor with another library such as tinymce, ckeeditor etc.

**Reply by ThoSap** [reply]
As you can see Quill is again under active development since September 2023.
The new v2.0 version, which will be released soon no longer has the mutation events which will be removed from Chrome/Chromium on July 2024 (Chromium 127).
https://github.com/quilljs/quill/issues/3806#issuecomment-1933250524
https://github.com/quilljs/quill/releases

I do not think the community will switch to TinyMCE, because for open-source it is limited to 1,000 editor loads per month and the CKEditor has many features behind a paywall, for example paste from Microsoft Word, which works in Quill v2.0.

**Reply by ThoSap** [reply]
See https://github.com/primefaces/primeng/issues/14721
